samples/java-protobuf-tracing/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+
+# If you're looking to use eventing with Google PubSub, to get an emulator running:
+# - add property "-Dkalix.proxy.eventing.support=google-pubsub-emulator" to the JAVA_TOOL_OPTIONS environment map under the kalix-runtime service
+# - uncomment the env var PUBSUB_EMULATOR_HOST and the section below for gcloud-pubsub-emulator service
+version: "3"
+services:
+  kalix-runtime:
+    image: gcr.io/kalix-public/kalix-runtime:1.1.34
+    container_name: tracing
+    ports:
+      - "9000:9000"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      JAVA_TOOL_OPTIONS: >
+        -Dkalix.proxy.telemetry.tracing.enabled=true
+        -Dkalix.proxy.telemetry.tracing.collector-endpoint=http://jaeger:4317
+      USER_SERVICE_HOST: ${USER_SERVICE_HOST:-host.docker.internal}
+      USER_SERVICE_PORT: ${USER_SERVICE_PORT:-8080}
```

Comment:
Check out how we are now making the docker-compose file. Variables go to .env file.

Reply:
.env added.